test/parallel/test-buffer-sharedarraybuffer.js

```diff
@@ -0,0 +1,26 @@
+/*global SharedArrayBuffer*/
+/*eslint no-undef: "error"*/
```

Comment:
The second line isn't necessary but the first one is absolutely required or else the linter fails.